src/main/java/net/mcreator/ui/modgui/LivingEntityGUI.java

```diff
@@ -645,6 +651,15 @@ private synchronized void regenerateAITasks() {
 		pane2.setOpaque(false);
 		pane2.add("Center", PanelUtils.totalCenterInPanel(spo2));
 
+		JComponent component = PanelUtils.northAndCenterElement(

```

Comment:
The name `component` sounds too common, try using something more unique.

Reply:
sure, changed it to layerList